api/src/test/java/org/apache/iceberg/expressions/TestAggregateBinding.java

```diff
@@ -57,6 +57,17 @@ public void testCountStarBinding() {
         .isEqualTo(Expression.Operation.COUNT_STAR);
   }
 
+  @Test
+  public void testCountNullBinding() {
```

Comment:
nit: instead of adding this test, let's add countNull to [`LIST`](https://github.com/apache/iceberg/blob/87fd6b388feed309d282aa9cd6b289d9d3bef7c4/api/src/test/java/org/apache/iceberg/expressions/TestAggregateBinding.java#L32), so it will be tested in `testAggregateBinding()`.